index.html

```diff
@@ -14,21 +14,115 @@
         <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
       </nav>
     </header>
-    <section id="intro">
-      <h1>I'm Farida Glad to see you!</h1>
-      <p id="bio">I’m a software developer! I can help you build a product, 
-        feature or website Look through some of my work and experience! 
-        If you like what you see and have a project you need coded, 
-        don’t hestiate to contact me.</p>
-      <h3>let's connect</h3>
-      
-      <ul id="social">
-        <li><img src="./images/Vector.png" alt="twitter"></li>
-        <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
-        <li><img src="./images/Vector.jpg" alt="email"></li>
-        <li><img src="./images/Vector-2.jpg" alt="github"></li>
-        <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
-      </ul>
-    </section>
+    <main>
+      <section id="intro">
+        <h1>I'm Farida Glad to see you!</h1>
+        <p id="bio">I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+        
+        <h3>let's connect</h3>
+        <ul class="social">
+          <li><img src="./images/Vector.png" alt="twitter"></li>
+          <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
+          <li><img src="./images/Vector.jpg" alt="email"></li>
+          <li><img src="./images/Vector-2.jpg" alt="github"></li>
+          <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
+        </ul>
+      </section>
+
+      <section class="previous-work">
+        <section class="project">
+          <img src="./images/Snapshoot-Portfolio.png" alt="project 1 snapshot">
+          <h1>Tonic</h1>
```

Comment:
- [ ] [REQUIRED] Please, always use the `<h1>` tag only 1️⃣ once in the entire document 📄. Since [it represents the title of the document](https://html.com/tags/heading/).